platform/viewer/src/components/Labelling/LabellingManager.js

```diff
@@ -1,21 +1,16 @@
 import React, { Component } from 'react';
```

Comment:
I assume the LabellingManager wraps the different labeling components and orchestrates their "flow"? 

If so, does it only render the `<LabelingFlow>` component? I think that would make this a container component we're trying to only have business logic in?

Reply:
Yeah, looks like it orchestrates to show edit description dialog or the labelling flow, I could level this code up to the provider and remove this component entirely.